app/assets/javascripts/admin/analytics-modules/ga4-finder-setup.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+window.GOVUK.analyticsGa4.analyticsModules =
+  window.GOVUK.analyticsGa4.analyticsModules || {}
+;(function (Modules) {
+  Modules.analyticsModules = Modules.analyticsModules || {}
+
+  const excludedTypes = ['search', 'hidden']
```

Comment:
Why is type 'search' excluded here - are we tracking it some other way?
May be useful to put an inline comment here.